.editorconfig

```diff
@@ -146,7 +146,7 @@ dotnet_diagnostic.CS1587.severity = none
 dotnet_diagnostic.CS1574.severity = none
 dotnet_diagnostic.CS0419.severity = none
 dotnet_diagnostic.CS1723.severity = none
-dotnet_diagnostic.CS1711.severity = none
+dotnet_diagnostic.CS1711.severity = error
```

Comment:
Just remove this, the default is error
```suggestion
```

Reply:
That was the first thing that I tried, but it didn't result in an error (before I made the change to remove the offending line in source):
<img width="1212" alt="image" src="https://user-images.githubusercontent.com/40371649/226067123-acacd3e6-6dd8-4743-b97d-2c0fe999f6ff.png">

Perhaps because this is a compiler warning and not an analyzer?

By the way, setting dotnet_analyzer_diagnostic.severity = error doesn't turn all of the analyzers on and have them error out; it only affects those that are turned on by default: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-options#scope

Reply:
Ah, good to know. I do recall this being not very clear when I first added this stuff. Maybe we can move these into a separate section at least so it's easy to see which ones still need to be fixed?

Reply:
Happy to move it if you think it's unclear on those marked as "none" vs. "error" (or put a comment)

Reply:
I just don't want us to get to the point where we have like 5 left and have to scan through a huge list of "error" ones to try and pick them out.